lib/release.sh

```diff
@@ -107,6 +107,15 @@ if [ ! "$STAGE" == "" ]; then
     if [ ! -f "$pkgVersionInstallScript" ];
     then
     
+      # if target stage is production, release the package version
+      if [ "$STAGE" == "PROD" ]; then
+      
+        log "Set package version as released ..."
+
+        invokeCmd "sfdx force:package:update -i \"$SFDX_PACKAGE_VERSION_ID\" --noprompt --setasreleased"
```

Comment:
Hi @wadewegner, the command is close but needs to be `force:package2:version:update` instead of `force:package:update` since we are releasing the specific version. Thanks!

Reply:
@wadewegner I have an Enterprise "production" org I'm testing with tonight and will let you know how that goes. Creating the package versions takes a while so I'm waiting a lot =)